pom.xml

```
@@ -43,6 +43,7 @@
         <module>spring</module>
         <module>flux-dsl</module>
         <module>examples</module>
+        <module>karaf</module>
```

Please, move `karaf` module before `spring` module:

```suggestion
        <module>karaf</module>
        <module>spring</module>
        <module>flux-dsl</module>
        <module>examples</module>
```